src/service/routers/RollupRouter.ts

```diff
@@ -60,9 +70,130 @@ export class RollupRouter {
+                    .withMessage("user_id is a required value")
+                    .isEthereumAddress()
+                    .withMessage("user_id is not wallet address type")
+                    .bail(),
+                body("state")
+                    .exists()
+                    .withMessage("state is a required value")
+                    .not()
+                    .isEmpty()
+                    .withMessage("state is a required value")
+                    .isIn(["0", "1"])
+                    .withMessage(`state input type error ,Enter "0" for charge or "1" for discharge`)
+                    .bail(),
+                body("amount")
+                    .exists()
+                    .withMessage("amount is a required value")
+                    .not()
+                    .isEmpty()
+                    .withMessage("amount is a required value")
+                    .custom(Validation.isAmount)
+                    .withMessage("amount can only be numbers type string")
+                    .bail(),
+                body("timestamp")
+                    .exists()
+                    .withMessage("timestamp is a required value")
+                    .not()
+                    .isEmpty()
+                    .withMessage("timestamp is a required value")
+                    .isNumeric()
+                    .withMessage("timestamp can only be numbers")
+                    .bail(),
+                body("exchange_user_id")
+                    .exists()
+                    .withMessage("exchange_user_id is a required value")
+                    .not()
+                    .isEmpty()
+                    .withMessage("exchange_user_id is a required value")
+                    .bail(),
+                body("exchange_id")
+                    .exists()
+                    .withMessage("exchange_id is a required value")
+                    .not()
+                    .isEmpty()
+                    .withMessage("exchange_id is a required value")
+                    .bail(),
+                // body("signer")
+                //   .exists()
+                //   .withMessage("signer is a required value")
+                //   .not()
+                //   .isEmpty()
+                //   .withMessage("signer is a required value")
+                //   .bail(),
+                // body("signature")
+                //   .exists()
+                //   .withMessage("signature is a required value")
+                //   .not()
+                //   .isEmpty()
+                //   .withMessage("signature is a required value")
+                //   .bail(),
+            ],
+            this.postTransactionSave.bind(this)
+        );
     }
 
     private static async getHealthStatus(req: express.Request, res: express.Response) {
         return res.json("OK");
     }
+
+    /**
+     * POST /tx/record
+     * @private
+     */
+    private async postTransactionSave(req: express.Request, res: express.Response) {
```

Comment:
```suggestion
    private async postTransactionRecord(req: express.Request, res: express.Response) {
```
API 라우터와 연결된 함수명을 쓰는것이 읽기 좋습니다.